btap/src/preprocessing.py

```diff
@@ -108,6 +111,7 @@ def read_weather(path: str) -> pd.DataFrame:
     """
     # Load the data from blob storage.
     s3 = acm.establish_s3_connection(acm.settings.MINIO_URL, acm.settings.MINIO_ACCESS_KEY, acm.settings.MINIO_SECRET_KEY)
+    logger.info("%s read_weather s3 connection %s", s3)
```

Comment:
Two placeholders specified, but only one argument.

```suggestion
    logger.info("read_weather s3 connection %s", s3)
```